android/build.gradle

```
@@ -25,7 +25,8 @@ apply plugin: 'com.android.library'
 apply plugin: 'kotlin-android'
 
 android {
-    compileSdkVersion 33
+    namespace 'es.antonborri.home_widget'
```

People do it like this

```suggestion
    // Conditional for compatibility with AGP <4.2.
    if (project.android.hasProperty("namespace")) {
        namespace 'es.antonborri.home_widget'
    }
```